pkgs/applications/networking/cluster/kubelogin-oidc/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "kubelogin";
+  version = "1.23.0";
+
+  src = fetchFromGitHub {
+    owner = "int128";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0n94nx17c6ln2nd6d9yr93vc251y1xphq1wj2vzs4j2l8dqfyjpn";
+  };
+
+  subPackages = ["."];
+
+  vendorSha256 = "1dvrk6z6k66wawgb50n8hbgdd8fly399mlbgnvxi671vfi7lkz09";
+
+  postInstall = ''
+    ln -s $out/bin/kubelogin $out/bin/kubectl-oidc_login
```

Comment:
Does this conflict with the other kubelogin?

Reply:
Probably! Both apps ship a binary called `kubelogin`.  It's somewhat unlikely that a user will need to install both simultaneously, but if that is a concern perhaps we could rename this one entirely instead of adding the symlink?